tests/Makefile

```diff
@@ -3,7 +3,7 @@
 # To compile with PKCS11: add "-lpkcs11-helper" to LDFLAGS
 
 CFLAGS	?= -O2
-WARNING_CFLAGS ?= -Wall -W -Wdeclaration-after-statement -Wno-unused-function -Wno-unused-value
+WARNING_CFLAGS ?= -Wall -W -Wdeclaration-after-statement -Wunused
```

Comment:
Minor: `-Wall` will also set many `-Wunused-*` options. I don't think we need to set `-Wunused` explicitly, but it doesn't hurt to set it here.